samples/nextjs/package.json

```diff
@@ -88,6 +89,7 @@
     "yaml-loader": "^0.6.0"
   },
   "scripts": {
+    "prepare": "cd ./node_modules/@sitecore-jss/sitecore-jss-nextjs && npm link ../../react",
```

Comment:
Not great that this will be in users apps when they create a nextjs sample. There isn't a way for us to do this in the repo root package.json scripts (even if it is ugly)?

Reply:
Changed